src/layouts/Home/components/Patterns.tsx

```diff
@@ -69,7 +69,7 @@ export const Patterns: React.FC = () => {
           >
             <SectionCallout
               section="Patterns"
-              header=" UI Patterns for building Zendesk products efficiently"
+              header="Existing design solutions for a more consistent UX"
```

Comment:
@hzhu @james-sann - I'm curious why this changed? Also just fyi I wasn't looped in on this content in the beginning so would love any additional context. Is this how the team is defining a pattern?

Reply:
This came from the [buttons draft PR](https://github.com/zendeskgarden/website/pull/260/files#diff-fef5f8c3fdd869fc68a64da111007a8560b21a0a0db0ab106b9d21dd5dbc884fR72). @jzempel do you have context on this copy?

Reply:
The #260 draft PR that Henry mentions was the original PR for adding the "Patterns" pillar to the website – and the baseline for this PR. This tagline was set at Content Design's request.

Reply:
@diana-oum thanks for looking into it and providing the reasons. The latest changes have been pushed up. Feel free to re-open this comment for further discussion.